WebRoot/js/zimbraMail/share/view/htmlEditor/ZmHtmlEditor.js

```diff
@@ -1250,21 +1260,85 @@ ZmHtmlEditor.prototype.onBeforeRepaint = function(ev) {
     }
 };
 
+ZmHtmlEditor.prototype.showComDragArea = function() {
+	var compDragArea = document.querySelector('.CompDragArea');
```

Comment:
document.querySelector will return the first element with class .CompDragArea, will this function work as expected in case we have multiple tabs for compose open ?